README.md

```diff
@@ -58,44 +61,59 @@ The result object can be accessed also via *oMX.dom* property. The properties av
  - error.text
 
 The sources must have the same default namespace (if have at all).
-Prefix '_' is reserved to handle default namespace.
+Prefix '_' is reserved to handle the default namespace.
 IE doesn't allow replacement of the root node attributes.
 
-The package
+NodeJS
 ------
+The browser window objects' (*DOMParser, XPathEvaluator, XMLSerializer*) functionality is implemented by the *xpath, xmldom* modules. The sample requires also the *formidable* module.
 
-The following files are included:
+Install the modules by starting **NPM** from the directory where you have downloaded the package files:
```

Comment:
> from the directory where you have downloaded the package files

That means nobody will use this. It will have to be installable with simply `npm install ....`

Reply:
If _devDependencies_ for the tests are included in package.json then simply 'npm install' installs myriads of modules. Therefore I'd recommend a minimum install at first (the tests are optional):
> npm install xpath xmldom formidable

Reply:
When you add mergexml as a dependency to an app, it will not install devDependencies. I think it uses `npm install --production` internally in that case.

You can further minimize the number of files with the `files` property in package.json.

Reply:
I'd prefer to use install options:

a) NodeJS modules
npm install --prod
A require path different from the default can be specified with the MergeXML  'path' option.

b) browsers test
npm install --dev
To remove: npm prune --prod